app/src/main/cpp/Cylinder.cpp

```diff
@@ -28,6 +28,8 @@ namespace crow {
 // Ratio between world size and cylinder surface size.
 // It should match the values defined in WindowWidget.
 // 800px is the default window size for a 4m world size.
+// TODO: For now, this value remains the same even if the default window size changes
+//       because otherwise the default windows are not correctly mapped on the cylinder.
```

Comment:
what do you mean here with the "Default windows" ?

Reply:
This value remains the same even if the default window size changes to ensure that three windows at the default size cover 180° of the cylinder.

If I increase this value following the increase in default window size, those three windows cover more than half of the cylinder.

Reply:
Comment updated:

```java
// TODO: For now, this value remains the same even if the default window size changes
//       to ensure that three windows at the default size cover 180° of the cylinder
//       (which is not the case if we increase this value).
```